pkg/ipfix/ipfix_collector_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ipfix
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	ipfixcollect "github.com/vmware/go-ipfix/pkg/collector"
+)
+
+func TestNewIPFIXCollectingProcess(t *testing.T) {
+	input := &ipfixcollect.CollectorInput{
+		Address:  "127.0.0.1:0",
+		Protocol: "tcp",
+	}
+	cp, err := NewIPFIXCollectingProcess(*input)
+	assert.NoError(t, err)
+	go cp.Start()
+	defer cp.Stop()
+	assert.NotNil(t, cp.GetMsgChan())
+	assert.Equal(t, 0, int(cp.GetNumRecordsReceived()))
+	assert.Equal(t, 0, int(cp.GetNumConnToCollector()))
```

Comment:
Sorry, not related to your test code, but actually the whole CollectorInput struct itself is meaningless as it just acts as a proxy class, which is not even needed in Go.
I have created https://github.com/antrea-io/antrea/pull/4505 to completely remove the unneeded structs. Thus, only test code in ipfix_registry_test.go is needed.

Reply:
So should I remove ipfix_collector_test.go , ipfix_intermediate_test.go and ipfix_process_test.go from the PR or will this PR be merged only after #4505 ?